mlflow/utils/cli_args.py

```diff
@@ -91,7 +91,7 @@ def _resolve_env_manager(ctx, _, env_manager):
         return env_manager
 
     # Neither `--no-conda` nor `--env-manager` is specified
-    return _EnvManager.CONDA
+    return None
```

Comment:
In MLflow projects, we need to select `env_manager` later based on the project environment type (e.g. if the environment type is conda, set `env_manager`to conda).

Reply:
Sounds good!